geoportal/c2cgeoportal_geoportal/views/layers.py

```diff
@@ -68,10 +69,120 @@
 
 if TYPE_CHECKING:
     from c2cgeoportal_commons.models import main  # pylint: disable=ungrouped-imports.useless-suppression
+
+
 _LOG = logging.getLogger(__name__)
 _CACHE_REGION = get_region("std")
 
 
+class _BaseCallback:
+    def __init__(self, layer: "main.Layer"):
```

Comment:
Why not:

```suggestion
    def __init__(self, layer: main.Layer):
```

Reply:
In fact, it's not possible because it's requires to import `c2cgeoportal_commons.models.main` and he is not initialized :-)